code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -289,7 +289,7 @@
 	rage_power = (1-(X.health/X.maxHealth)) * RAVAGER_RAGE_POWER_MULTIPLIER //Calculate the power of our rage; scales with difference between current and max HP
 
 	if(X.health < 0) //If we're at less than 0 HP, it's time to max rage.
-		rage_power = 1
+		rage_power = 0.5
 
```

Comment:
Why is this if check needed if `(1 - (0 / maxHealth)) * 0.5` is 0.5 anyways?

Reply:
X.health wouldn't necessarily be 0 if you're enduring.